doc/tutorial/single_cell_detailed_recipe.rst

```diff
@@ -49,71 +49,78 @@ examine the recipe in detail: how to create one, and why it is needed.
 
 .. literalinclude:: ../../python/example/single_cell_detailed_recipe.py
    :language: python
-   :lines: 84-133
+   :lines: 84-132
 
 Let's go through the recipe point by point.
 
 Step **(5)** creates a ``single_recipe`` class that inherits from :class:`arbor.recipe`. The base recipe
 implements all the methods defined above with default values except :meth:`arbor.recipe.num_cells`,
 :meth:`arbor.recipe.cell_kind` and :meth:`arbor.recipe.cell_description` which always have to be implemented
 by the user. The :meth:`arbor.recipe.global_properties` also needs to be implemented for
-:class:`arbor.cell_kind.cable` cells. The inherited recipe can implement any number of additional methods and
-have any number of instance or class variables.
+:class:`arbor.cell_kind.cable` cells.
 
-Step **(5.1)** defines the class constructor. In this case, we pass a ``cell`` and a set of ``probes`` as
-arguments. These will be used to initialize the instance variables ``self.the_cell`` and ``self.the_probes``,
-which will be used in the overloaded ``cell_description`` and ``get_probes`` methods. Before variable
-initialization, we call the base C++ class constructor ``arbor.recipe.__init__(self)``. This ensures correct
-initialization of memory in the C++ class.
+.. note::
+   The inherited recipe can implement any number of additional methods and have any number of instance or class
+   variables. The structure shown in this example is a choice; it's up to you to structure your code the way you
+   see fit. What matters is that the methods required to be present return valid results of the correct type.
```

Comment:
This can be understood as that the overloads on all methods are required. Think of those less versed in the ways of OOP/Python.

Reply:
TIL you don't need to override methods you don't use in your model. I'm not sure it'll bring much clarify to make that distinction here though. The way I read it now implies all that's overloaded in this example needs to be overloaded always, which is a fine message to pass on imho, even if not 100% correct.